api_docs/changelog.md

```diff
@@ -20,6 +20,16 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 263**
+
+* [`GET /messages`](/api/get-messages),
+  [`GET /messages/matches_narrow`](/api/check-messages-match-narrow),
+  [`POST /messages/flags/narrow`](/api/update-message-flags-for-narrow),
+  [`POST /register`](/api/register-queue):
+  Added support for a [search/narrow filter](/api/construct-narrow)
+  related to followed topics: `is:followed`. This operator narrows to
+  messages from topics that the user has [followed](/help/follow-a-topic).
```

Comment:
I kinda wonder if this should just link to https://zulip.com/api/construct-narrow rather than the individual endpoints, since that's the part of the API that you'd be looking at to see what's changed. Probably worth an "api documentation" stream discussion, since we'd want to change the existing ones if so.

Reply:
I've opened a discussion at [CZO](https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/Narrow.20filter.20documentation).